### PR TITLE
feat: update firebase v1 integration

### PR DESCRIPTION
**Pull request by shwetalodhe123**
1. update firebase sdk version to 10.28.0 
2. generate privacy report after sdk update 
3. update pod minimum version to 12.0 those are below 12.0

### CODE REVIEW

**Comment by shwetalodhe123** [comment]
for testing realtime events on firebase we have added googleservice-info.plist. Later it is been discarded and push updated code to branch.

**Review by SKannaniOS — APPROVED** [review]
Since this PR is about updating the Firebase SDK version, everything looks good to me.

**Review by 1abhishekpandey — CHANGES_REQUESTED** [review]
Currently `pod install` command is failing:
image

To solve that execute the following steps:
1. Remove the `Podfile.lock` (type `rm Podfile.lock`).
2. Perform `pod install` (it'll generate the new `Podfile.lock` file with the updated details)
3. Push the changes.

File: Rudder-Firebase.podspec
Comment (1abhishekpandey) [inline Rudder-Firebase.podspec:1]: Rename the `license` file by changing [this](https://github.com/rudderlabs/rudder-integration-firebase-ios/blob/6e206526c4f82e15133fdb10ba8c32a8bf23743c/Rudder-Firebase.podspec#L20) line to:
```
s.license          = { :type => "Apache", :file => "LICENSE.md" }
``` 

It'll solve the following issue while performing `pod install`:
image

**Review by 1abhishekpandey — CHANGES_REQUESTED** [review]
There is a stale reference of the `GoogleService-Info` file:
image

Please remove that.

File: Rudder-Firebase.podspec
Comment (shwetalodhe123) [reply Rudder-Firebase.podspec:1]: Done

**Comment by shwetalodhe123** [comment]
Unable to see deleted file GoogleService-Info in project, seems deleted at my end

**Comment by shwetalodhe123** [comment]
> Currently `pod install` command is failing: image
> 
> To solve that execute the following steps:
> 
> 1. Remove the `Podfile.lock` (type `rm Podfile.lock`).
> 2. Perform `pod install` (it'll generate the new `Podfile.lock` file with the updated details)
> 3. Push the changes.

Done

**Review by 1abhishekpandey — APPROVED** [review]

**Pull request closed without merging**